content/includes/business-logos.html

```diff
@@ -0,0 +1,19 @@
+<section id="business-logo">
+    <div class="ui very padded container">
+        <div class="ui four column grid">
+            <div class="column">
```

Comment:
The logos appear out of alignment in part because they are top aligned inside the column. You can set the column to middle alignment with:

```suggestion
            <div class="middle aligned column">
```